### DISCUSSION TITLE
how to split window with other profile?

### DISCUSSION THREAD

**Discussion by sdjeong-zenaad** [Eugeny/tabby#7008] [Q&A] [answered]
I want to split pane with other profile.
But, it can only split same profile.
How to split for new profile?

**Comment by Eugeny** [comment, marked as answer]
You can either: 
a) open two tabs and drag one tab beside another
b) split one tab with the same profile and then right click -> switch profile in one of the panes

**Reply by sdjeong-zenaad** [reply]
thank you very much!
a) is not work well, but b) is good.

**Reply by lawweiliang** [reply]
@sdjeong-zenaad 
Solution B with hotkey.
Window: Ctrl+Alt+T      😄 


---
Ctrl + Shift + [X]  --> For Tab
Ctrl + Alt + [X]   --> For Pane

**Comment by lawweiliang** [comment]
@sdjeong-zenaad 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
